StoreKitUnitTests/ProductDetailsTests.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  ProductDetailsTests.swift
+//
+//  Created by Andrés Boedo on 1/9/21.
+
+import Nimble
+@testable import RevenueCat
+import StoreKitTest
+import XCTest
+
+class ProductDetailsTests: XCTestCase {
+
+    var testSession: SKTestSession!
+    var userDefaults: UserDefaults!
+
+    override func setUpWithError() throws {
+        testSession = try SKTestSession(configurationFileNamed: "UnitTestsConfiguration")
+        testSession.disableDialogs = true
+        testSession.clearTransactions()
+    }
+
+    @available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
+    func testSK1AndSK2DetailsAreEquivalent() async throws {
+        guard #available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *) else { return }
```

Comment:
is this really needed?

<img width="1359" alt="Screen Shot 2021-09-21 at 3 15 12 PM" src="https://user-images.githubusercontent.com/664544/134254930-dad19225-491c-45e9-8bbc-69ab99cec98d.png">

Reply:
yes. although it's dumb. 

`@available` doesn't work with test cases for reasons that I cannot comprehend. So even if you set it, older iOS versions will still try to execute the test, and crash. 

However, we still need to set `@available` in the first place, otherwise the compiler would complain because we're using `async` in the method signature, which is only available in iOS 15+. 

https://stackoverflow.com/questions/59645536/available-attribute-does-not-work-with-xctest-classes-or-methods

https://www.wwdcnotes.com/notes/wwdc20/10164/

I would absolutely love to find alternatives, though, since this produces that super annoying warning. Also, it looks dumb.

Reply:
Oh that's definitely dumb lol